### ISSUE TITLE
Remove green checkmark from docs Markdown

### ISSUE THREAD

**Issue by hpretl** [efabless/cace#108]
There is a green checkmark in the summary section of the result Markdown. This special characters fails in some environments, and it is not really needed in my view. Please just stick with a "Pass"/"Fail" verdict.

**Comment by hpretl** [comment]
@mole99 @RTimothyEdwards Please take a look if you agree.

**Comment by mole99** [comment]
Hi Harald, can you tell me under which environment it fails?

I personally like the green checkmark ✅ and the red cross ❌ to see at a glance if a parameter failed or everything was successful.

OpenLane 2 already makes heavy use of Unicode and ANSI escape codes and any modern terminal emulator should have support for this. (You too now Tim 😉)

**Comment by hpretl** [comment]
It fails when compiling the `md` with Quarto/Pandoc into Latex to compile to a PDF :-) Sounds complicated, but I use it to do this: https://iic-jku.github.io/analog-circuit-design/#t-ota-simulation-versus-pvt

**Comment by hpretl** [comment]
The alternative is that I do post-processing of the Md to remove these characters, as I do currently with the figures paths (need to tweak the paths). Do you have any idea how to do this with a `sed`?

**Comment by mole99** [comment]
@RTimothyEdwards Yes, we could try to use alternative characters. How about 🮱 or ✓ and 🮽 or ⨯ ?

**Comment by hpretl** [comment]
Whatever these characters are, they do not even render correctly in my browser :-( 

<img width="228" alt="image" src="https://github.com/user-attachments/assets/eefbf45d-59e4-4be2-ac66-b80dbc61ab7e">

**Comment by mole99** [comment]
It should look like: 

![grafik](https://github.com/user-attachments/assets/aa40c2ec-af39-4a7c-8ed2-ae54fc40563e)


That is really strange, they seem to be basic Unicode symbols. Does your terminal render them correctly?

**Comment by hpretl** [comment]
I changed the Latex compiler to Xelatex, and now I can render w/o fail, although it still does not show this caracter. I use now a script to remove all unicode characters :-)

**Comment by hpretl** [comment]
Not sure about terminal, but also my mail program can not render it (all macOS).

**Comment by mole99** [comment]
Your browser and mail program probably use the same font which does not support these characters. Are you using the default macOS font? If so, then I'm a bit surprised that they're not supported, but ✅ and ❌ both work?

**Comment by RTimothyEdwards** [comment]
@hpretl :  Just be aware that beyond the check-mark and X-mark, unicode symbols are being used for basic engineering text like square root (as in 1/sqrt(Hz)), squared, micro, degree, and a handful of others.  Some of those are in the ISO font encoding and some in the standard symbol font, but I don't know how universally they display.  But just watch out for the possibility that wiping all unicode out of the output will screw up a lot of units.

**Comment by hpretl** [comment]
@RTimothyEdwards Good point! Maybe I need to selectively remove Unicode characters then. But so far, I avoid Unicode wherever I can, and use things like uV and rtHz. :-)
